bash_completion

```diff
@@ -504,14 +536,31 @@ _comp_compgen()
             return 2
         fi
 
+        if [[ $_dir ]]; then
+            local _original_pwd=$PWD
+            local PWD=$PWD OLDPWD=$OLDPWD
```

Comment:
After another thinking, I would like to keep these values. If we do not initialize `PWD`, `OLDPWD` becomes unset after `cd` (unless `localvar_inherit` is set). If we use `cd -` later, it fails. Even when we do not use `cd -`, an unset OLDPWD doesn't seem to be a desirable state. The unset OLDPWD may cause problems in a generator that references OLDPWD, though I'm not sure if there is an actual case.

On the other hand, `OLDPWD` can be empty here because it will soon be initialized by the succeeding `cd`, but initializing one of the two in `local` doesn't look neat.